libosrmc/osrmc.h

```diff
@@ -1,3 +1,5 @@
+#include <stdbool.h>
```

Comment:
stdbool is :+1: C99 guarantees for this header. And if you need pre-C99 you are sorta on your own anyway.